sfdx-project.json

```diff
@@ -27,5 +27,13 @@
   "packageAliases": {
     "Apex Mockery": "0HoDn000000kA5aKAE",
     "Apex Mockery NS Test": "0HoDn000000kA6JKAU"
-  }
+  },
+  "replacements": [
+    {
+      "glob": "force-app/recipes/**/*.cls",
+      "stringToReplace": "namespace.",
+      "replaceWithEnv": "SF_NAMESPACE_DOT",
```

Comment:
**question:** `SF_NAMESPACE_DOT` should be used then in the github actions right ?

We should remove then this [githubaction](https://github.com/salesforce/apex-mockery/blob/main/.github/workflows/validate-namespace-compatibility.yml)
And only use this [one](https://github.com/salesforce/apex-mockery/blob/main/.github/workflows/validate-package-version.yml):
- It should take a parameter a boolean "namespace". When namespace is true it should set the `SF_NAMESPACE_DOT` with the value `mockery.`, else it should set it with empty string
- this [github action](https://github.com/salesforce/apex-mockery/blob/main/.github/workflows/on-main-push.yml) should be modified to call the only one action twice with the "namespace" boolean true for `validate-namespace-compatibility` step and false for `validate-package-version`

Something like that ?

Reply:
That's exactly it. It should also give people the opportunity to use their own namespace if they want to fork the pacakge.